frontend/app/services/EventbriteService.scala

```diff
@@ -143,7 +143,7 @@ object GuardianLiveEventService extends LiveService {
 
 object LocalEventService extends LiveService {
   val apiToken = Config.eventbriteLocalApiToken
-  val maxDiscountQuantityAvailable = 2 //TODO are these discounts correct for local?
+  val maxDiscountQuantityAvailable = 3
```

Comment:
In order to QA free event tickets we rely on the Eventbrite dev account, which is configured to be  handled through the `LocalEventService`